doc/api/console.md

```diff
@@ -9,10 +9,14 @@ The module exports two specific components:
 
 * A `Console` class with methods such as `console.log()`, `console.error()` and
   `console.warn()` that can be used to write to any Node.js stream.
-* A global `console` instance configured to write to `stdout` and `stderr`.
-  Because this object is global, it can be used without calling
+* A global `console` instance configured to write to [`process.stdout`][] and
+  [`process.stderr`][].  This object is global, it can be used without calling
   `require('console')`.
 
+***Warning***: The global console object's methods may be synchronous, because
```

Comment:
Better yeah, wording reads a bit weird to me but idk how to improve it atm. Something regarding the `neither`.